game/src/devtools/fifteen_min/mod.rs

```diff
@@ -38,6 +38,26 @@ impl Viewer {
     pub fn new(ctx: &mut EventCtx, app: &App, start: BuildingID) -> Box<dyn State<App>> {
         let start = app.primary.map.get_b(start);
         let isochrone = Isochrone::new(ctx, app, start.id);
+
```

Comment:
I think this is leftover from the merge; everything from here down to `rows.push(txt.draw(ctx));` can be reverted